src/main/java/org/broadinstitute/hellbender/utils/haplotype/BiallelicVariant.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.utils.haplotype;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.utils.Utils;
+
+/**
+ * Very simple class wrapping VariantContext when we want to be explicit that a variant is biallelic, such as
+ * in a haplotype EventMap
+ */
+public class BiallelicVariant extends VariantContext {
+    private static final long serialVersionUID = 1L;
+
+    public BiallelicVariant(final VariantContext vc) {
+        super(vc);
```

Comment:
This is a good idea for a fix... I ALMOST want to suggest we disentangle this from all of the heavy VCF construction validation steps in HTSJDK that we usually undergo when we call the VCF constructor... those can actually show up as a non-insignificant amount of the runtime in profilers in some circumstances....

Reply:
I'm also tempted.  It's doable to replace this with a class (maybe call it `Event` to parallel `EventMap`?) that contains just a ref, alt, and position.  Think I should?

Reply:
I think in a lot of ways its cleaner... I like Event <-> EventMap... it could also have an event.asVCFLine() argument to convert it more lazily when we actually need a vcf in the code... It would mean pulling a lot of functionality that the htsjdk vcf was previously handing into this class however